include/libp2p/layer/websocket/ws_connection.hpp

```diff
@@ -6,26 +6,31 @@
 #ifndef LIBP2P_CONNECTION_WSCONNECTION
 #define LIBP2P_CONNECTION_WSCONNECTION
 
+#include <boost/beast/websocket.hpp>
```

Comment:
We separate internal includes, std libs includes and external
```suggestion
#include <boost/beast/websocket.hpp>

```